libra-swarm/src/swarm.rs

```diff
@@ -267,7 +267,7 @@ impl LibraSwarm {
     /// assumably due to previous launch failure, it will be removed.
     /// The directory for the last failed attempt won't be removed.
     fn setup_config_dir(config_dir: &Option<String>) -> LibraSwarmDir {
-        let dir = match config_dir {
+        match config_dir {
```

Comment:
For commits like this, it's best to use `git commit --fixup` so they get auto-squashed later and will not appear in the repo history.

Reply:
In general, I agree, but this complaint had nothing to do with any of my commits. So it would be unclear which to associate it with / fixup.